### PR TITLE
Update microsoft-teams-online-call-flows.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Updates a couple of markdowns. 
Corrected a few URLs.
General "tidying up" to follow markdown style rules.
Edited lists to follow number styling from guidelines.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: Teams/microsoft-teams-online-call-flows.md [hunk 1]
```diff
@@ -311,11 +328,12 @@ Note that:
...
+To support this scenario, the customer must deploy a certified SBC for Direct Routing from one of Microsoft's certified partners. The SBC must be configured as recommended by the vendor, and be routable from Office 365 for direct UDP traffic. The media may flow directly from Teams and/or the Skype for Business client to the SBC (bypassing the Teams gateway) or traverse through the Teams gateway. The connectivity with the SBC, when the trunk is configured to bypass the Teams gateway, is based on ICE, where SBC supports ICE-Lite, while the Teams/Skype for Business media endpoint supports ICE Full.
```
Comment (JohanFreelancer9) [inline Teams/microsoft-teams-online-call-flows.md]:
```suggestion
To support this scenario, the customer must deploy a certified SBC for Direct Routing from one of Microsoft's certified partners. The SBC must be configured as recommended by the vendor, and be routable from Office 365 for direct UDP traffic. The media may flow directly from Teams and/or the Skype for Business client to the SBC (bypassing the Teams gateway) or traverse through the Teams gateway. The connectivity with the SBC, when the trunk is configured to bypass the Teams gateway, is based on ICE, where SBC supports ICE-Lite, while the Teams/Skype for Business media endpoint supports ICE Full Form.
```

File: Teams/microsoft-teams-online-call-flows.md [hunk 2]
```diff
@@ -446,10 +464,10 @@ Note that:
 [![Microsoft Teams Online Call Flows Figure 25](media/microsoft-teams-online-call-flows-figure25-thumbnail.png)](media/microsoft-teams-online-call-flows-figure25.png)
 
 *Figure 25 - Teams with Express Route optimization*
- 
-In the case that Express Route is justified and deployed, then Teams flows could be re-routed from flow 4 to flow 1 and from flow 4' to flow 1'. However, Teams Application has a hard dependency on other Office 365 flows over the internet via flows 4 and 4'; hence these flows must not be blocked. 
 
-Note that Skype for Business hybrid Edge traffic is routed to the Internet and not to Express Route to communicate with external users and federate with other tenants. 
+In the case that Express Route is justified and deployed, then Teams flows could be re-routed from flow 4 to flow 1 and from flow 4' to flow 1'. However, Teams Application has a hard dependency on other Office 365 flows over the internet via flows 4 and 4'; hence these flows must not be blocked.
```
Comment (JohanFreelancer9) [inline Teams/microsoft-teams-online-call-flows.md]:
```suggestion
In the case that Express Route is justified and deployed, then Teams flows could be re-routed from flow 4 to flow 1 and from flow 4' to flow 1'. However, the Teams application has a hard dependency on other Office 365 flows over the Internet via flows 4 and 4'; hence these flows must not be blocked.
```

File: Teams/microsoft-teams-online-call-flows.md [hunk 3]
```diff
@@ -460,15 +478,17 @@ For example:
...
-4. OFFICE 365 sends notification to external Teams user via flow 3
-5. Teams external user allocates a media Relay port on Teams Transport Relay via flow 3
-6. Teams external user sends "answer" with ICE candidates via flow 3, which is forwarded back to Teams user A via Flow 1
-7. Teams User A and Teams User B invoke ICE connectivity tests and selects flows 1 and 3, which are relayed by Teams Transport Relay in Office 365
-8. Teams Users send telemetry to Office 365 via flows 1 and 3
-
->**Note**: Flow 4 must be enabled to support dependencies of Teams application on other micro-services that mandates flow 4.
+1. Teams User within customer network allocates a media Relay port on Teams Transport Relay via flow 1
+1. Teams User within customer network sends "invite" with ICE candidates via flow 1 to Office 365
+1. OFFICE 365 sends notification to external Teams user via flow 3
```
Comment (JohanFreelancer9) [inline Teams/microsoft-teams-online-call-flows.md]:
```suggestion
1. Office 365 sends notification to external Teams user via flow 3
```

**Comment by baardhermansen** [comment]
> Copy Review - just a few minor suggestions.

Well spottet, i did not pay enough attention...

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**